### PR TITLE
Lma 152

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: VAS.Core/StyleConf.cs [hunk 1]
```diff
@@ -98,6 +94,10 @@ public StyleConf ()
 		public Color ColorAccentSuccess { get; set; }
 		public Color ColorAccentWarning { get; set; }
 		public Color ColorAccentError { get; set; }
+		public Color ColorFullWhite { get; set; }
+		public Color ColorShadow { get; set; }
+
+		public Color DrawingSelectorBorder { get; set; }
```
Comment (vguzmanp) [inline VAS.Core/StyleConf.cs]: All start with `Color` except for this one

File: VAS.Core/StyleConf.cs [hunk 2]
```diff
@@ -98,6 +94,10 @@ public StyleConf ()
 		public Color ColorAccentSuccess { get; set; }
 		public Color ColorAccentWarning { get; set; }
 		public Color ColorAccentError { get; set; }
+		public Color ColorFullWhite { get; set; }
```
Comment (vguzmanp) [inline VAS.Core/StyleConf.cs]: This shouldn't include the color name.

Comment (ylatuya) [reply VAS.Core/StyleConf.cs]: It must include it because it's an absolute white, independent of the theme

File: VAS.Core/StyleConf.cs (hunk 1 above)
Comment (ylatuya) [reply VAS.Core/StyleConf.cs]: This color was already in the palette, I have left it as it is

File: VAS.Core/StyleConf.cs (hunk 2 above)
Comment (vguzmanp) [reply VAS.Core/StyleConf.cs]: Then it doesn't make sense to have it in the palette.

Why do we need absolute white, independent of the theme?

**Pull request merged**